spec/factories/shopify_api/inventory_level_factory.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :shopify_inventory_level, class: ShopifyAPI::InventoryLevel do
+    skip_create
+    inventory_item_id   { rand(0..1000) }
```

Comment:
why do you need to randomize `id`?

may be you can set it to any number and that's it?

Reply:
👍